src/Security/Security.php

```diff
@@ -1073,7 +1073,7 @@ public function getTemplatesFor($action)
      *
      * @return Member
      *
-     * @deprecated 4.0.0:5.0.0 Please use DefaultAdminService::findOrCreateDefaultAdmin()
+     * @deprecated 4.0.0 Please use DefaultAdminService::findOrCreateDefaultAdmin()
```

Comment:
Update the notice message and version as well.
```suggestion
     * @deprecated 4.0.0 Use DefaultAdminService::findOrCreateDefaultAdmin() instead
```